### PR TITLE
Add server and ml model, create db model

### PR DESCRIPTION
**Pull request by krydev**
## Add server, serve ML model, create db model

:point_up: Single pull request should have less than 600 lines of new code (excluding code of frameworks or third-party libraries)
:point_up: For pull request create your own branch <your_login>/<name_of_branch>
:point_up: Base branch - develop
:point_up: After pull request will be merged, please, delete branch of pull request

#### Type of changes (please mark pull request with corresponding label)
- [ ] New feature
- [ ] Bug
- [ ] Client
- [x] Server
- [x] Model
- [x] DB

#### What's new
- Created server with endpoint `/predict` to upload an image and get ML model results
- Created ML model to classify images
- Created DB model for prediction records

### CODE REVIEW

**Review by akiya-nagatsuka — APPROVED** [review]

File: server/ml_model/model_base.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+from PIL import Image
+from torchvision import models
+
+imagenet_class_index = json.load(open('ml_model/imagenet_class_index.json'))
+model = models.densenet121(pretrained=True)
+model.eval()
+
+
+def transform_image(image_bytes):
+    my_transforms = transforms.Compose([transforms.Resize(255),
```
Comment (livankrekh) [inline server/ml_model/model_base.py:14]: Please, don't use magic numbers! Other team members should know, what means number 255 in your code.
Please, add new variable with appropriate name and will be great if you add comment to variable

File: server/ml_model/model_base.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+                                        transforms.CenterCrop(224),
```
Comment (livankrekh) [inline server/ml_model/model_base.py:15]: The same as before

File: server/ml_model/model_base.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+                                        transforms.ToTensor(),
+                                        transforms.Normalize(
+                                            [0.485, 0.456, 0.406],
```
Comment (livankrekh) [inline server/ml_model/model_base.py:18]: The same as before

**Review by livankrekh — APPROVED** [review]
See my minor comments and fix it in next PRs

File: server/ml_model/model_base.py (hunk 1 above)
Comment (krydev) [reply server/ml_model/model_base.py:14]: Thank you for the code review! You're absolutely right.
The thing is, I just copy-pasted this piece of code, so I have no idea what those numbers mean.

**Review by ivgnt — APPROVED** [review]
